<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nodeColorize</class>
 <widget class="QMainWindow" name="nodeColorize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pw Node Colorize</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="get_btn">
          <property name="text">
           <string>&gt; GET</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="set_btn">
          <property name="text">
           <string>SET &gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Color Ramp</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="picker_ly"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
